src/Protocol/Salmon.php

```diff
@@ -201,4 +200,14 @@ public static function slapper($owner, $url, $slap)
 
 		return (($return_code >= 200) && ($return_code < 300)) ? 0 : 1;
 	}
+
+	/**
+	 * @param string $pubkey public key
+	 * @return string
+	 */
+	public static function salmonKey($pubkey)
```

Comment:
I wanted to ask you to rename this function with its actual function, but I'm actually unable to figure out what it is doing exactly, so I guess we'll leave it at that 😂

Reply:
It encodes the public key of the user that is needed for Salmon notifications.

Reply:
So `Salmon::encodePubKey`?

Reply:
Doesn't sound worse than the existing name.

Reply:
I appreciate your enthusiastic support!